charts/scalardb-analytics-postgresql/values.yaml

```diff
@@ -0,0 +1,163 @@
+    databaseName: scalardb
+    # -- The secret resource name that includes superuser password for PostgreSQL.
+    secretName: scalardb-analytics-postgresql-superuser-password
+
+  # --  The database.properties to access the underlying databases that Schema Importer will load schemas into PostgreSQL.
+  # @default -- The minimum template of database.properties is set by default.
+  databaseProperties: |
+    # Comma separated contact points. For DynamoDB, the region is specified by this parameter.
+    scalar.db.contact_points=${env:SCALAR_DB_CONTACT_POINTS:-}
+
+    # Port number for all the contact points. Default port number for each database is used if empty.
+    scalar.db.contact_port=${env:SCALAR_DB_CONTACT_PORT:-}
+
+    # Credential information to access the database. For Cosmos DB, username isn't used. For DynamoDB, AWS_ACCESS_KEY_ID is specified by the username and AWS_ACCESS_SECRET_KEY is specified by the password.
+    scalar.db.username=${env:SCALAR_DB_USERNAME:-}
+    scalar.db.password=${env:SCALAR_DB_PASSWORD:-}
+
+    # Storage implementation. "cassandra" or "cosmos" or "dynamo" or "jdbc" or "grpc" can be set. Default storage is "cassandra".
+    scalar.db.storage=${env:SCALAR_DB_STORAGE:-}
+
+  # -- Secret name that includes sensitive data such as credentials. Each secret key is passed to Pod as environment variables using envFrom.
+  secretName: ""
+
+  image:
+    # -- Docker image repository of ScalarDB Analytics with PostgreSQL.
+    repository: ghcr.io/scalar-labs/scalardb-analytics-postgresql
+    # -- Specify a image pulling policy.
+    pullPolicy: IfNotPresent
+    # -- Override the image tag whose default is the chart appVersion
+    tag: ""
+
+  # -- Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace.
+  imagePullSecrets: [name: reg-docker-secrets]
+
+  strategy:
+    rollingUpdate:
+      # -- The number of pods that can be created above the desired amount of pods during an update
+      maxSurge: 25%
+      # -- The number of pods that can be unavailable during the update process
+      maxUnavailable: 25%
+    # -- New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate
+    type: RollingUpdate
+
+  service:
+    # -- service types in kubernetes
+    type: ClusterIP
+    # -- Service annotations, e.g: prometheus, etc.
+    annotations: {}
+    ports:
+      postgresql:
+        # -- PostgreSQL public port
+        port: 5432
+        # -- PostgreSQL k8s internal port
+        targetPort: 5432
+        # -- PostgreSQL protocol
+        protocol: TCP
+
+  # -- PodSecurityContext holds pod-level security attributes and common container settings.
+  podSecurityContext:
+    # -- To work ScalarDB Analytics with PostgreSQL properly, you must set "201" to "podSecurityContext.fsGroup".
+    fsGroup: 201
```

Comment:
To run the `entorypoint.sh` by `scalar (UID=201)` user in the Schema Importer container, we have to mount the `entrypoint.sh` file with `201:201` configuration as a file owner configuration. So, we have to set `fsGroup=201` here.